.github/workflows/build.yml

```diff
@@ -70,33 +74,25 @@ jobs:
       - name: Install necessary dependencies
         run: |
           pip install pyyaml
-          pip install git+https://github.com/eduNEXT/tvm.git
 
       - name: Get Tutor Configurations from config.yml and set them as an environment variable
+        working-directory: ${{ github.workspace }}
         env:
-          REQUIRED_KEYS: TUTOR_VERSION TUTOR_APP_NAME
+          REQUIRED_KEYS: TUTOR_VERSION
```

Comment:
If we no longer require this variable, we should, **as future work**, remove that variable from the documentation of PICASSO: https://internal.docs.edunext.co/en/latest/internal/Products_and_Services/by_products/picasso/picasso-v2.html?highlight=picasso%20v2#requirements

Reply:
I would say let's do it as soon as possible so we always keep the documentation updated